benchopt/tests/test_benchmarks/dummy_benchmark/objective.py

```diff
@@ -38,7 +38,9 @@ def skip(self, X, y):
     def get_one_solution(self):
         return np.zeros(self.X.shape[1])
 
-    def evaluate_result(self, beta):
+    def evaluate_result(self, solver_result):
+        beta = solver_result["beta"]
```

Comment:
I don't see how this fixes the issue you mention, are all files committed?

Reply:
I'm not sure this setting should happen: the objective no longer has a compute method, so we expect the benchmark has been updated, and evaluate_result is correctly implemented (takes an unpacked dict as input). 

Do you confirm @Badr-MOUFAD ?

Reply:
> I don't see how this fixes the issue you mention, are all files committed?

It wasn't actually a fix, it was to point out where it would break

> I'm not sure this setting should happen: the objective no longer has a compute method, so we expect the benchmark has been updated, and evaluate_result is correctly implemented (takes an unpacked dict as input).

Aren't we going through a deprecation cycle? In fact, #576 breaks the benchmark passing in dict to compute.